### DISCUSSION TITLE
Multiple BT inverters ?

### DISCUSSION THREAD

**Discussion by lebeck69** [SBFspot/SBFspot#588] [Q&A]
I've seen the SBFspot is working with multiple IP inverters. (coma separated)
But is it with bluetooth ones ?

I have 2 SB 3000HF-30 
I'm using home assistant
HasQT kindly created a second instance MultiINV-SBFspot as a work around but I wonder if this can be included.
and top of the top would be to add both inverter production info before going to pvoutput ?!

I'm new to all this.
Already thx for reading up to here ^^

**Comment by sbf-** [comment]
Sure this is possible:

Make sure NetID of all your inverters is set at 2 or higher
Set MIS_enabled=1 in SBFspot.cfg

SMA BT inverters build their own network. SBFspot connects to 1 inverter and is able to communicate with all the other inverters

**Comment by lebeck69** [comment]
Thx for you reply

Don't know if I've done something wrong

I ve tryed MIS option and it didn't worked for me. 
I have SMA inverter 1 netID=04 & inverter 2 SMA netID=04
Only retrieving data of first inverter.

**Reply by sbf-** [reply]
Pls provide debug log (-d5 -v5)

**Reply by habuild** [reply]
For reference this is issue coming from [haos-sbfspot home assistant addon issue 42](https://github.com/habuild/hassio-addons/issues/42)

posting for future people's

Thanks as always for SBFspot!
